### PR TITLE
Remove the buttons from the center of the page and place them as links in navbar

### LINKED ISSUE #221: Remove the buttons from the center of the page and have them as links on the navbar
Remove the buttons from the center of the page and have them as links on the navbar

### PR DESCRIPTION
**Pull request by subhangi2731**
Removed the buttons from the center of the page and placed them as links in the navbar. Also removed the logo and placed it in the navbar. All the contents from the body of the main page are also removed as the calendar has to be integrated and set up with the search meeting logs bar for the recent meetings

Closes #221

### CODE REVIEW

**Comment by subhangi2731** [comment]
@t0xic0der @darknao please have a look at this and let me know

**Comment by darknao** [comment]
Please add the `.png` extension to the logo file. Maybe something went wrong with this commit, as this file is actually empty ?
And buttons are still here in the center of the page.

**Comment by subhangi2731** [comment]
@darknao oh sorry let me check once again

**Comment by subhangi2731** [comment]
@darknao any more changes to be done?

File: src/mote/templates/mainpage.html [hunk 1]
```diff
@@ -51,18 +52,31 @@
...
+                    type="button"
+                    onclick="populate_channel_list();">
+                    Meetings by Channel
+                    </a>
+                    <a class="nav-link"
+                    aria-current="page"
+                    data-bs-toggle="modal"
+                    data-bs-target="#chanmode"
+                    type="button"
+                    onclick="populate_recent_meeting_list();">
```
Comment (darknao) [inline src/mote/templates/mainpage.html:71]: I think we can remove the `onclick` attribute for now, as the `meetings by group` page doesn't exist yet.

**Review by darknao — CHANGES_REQUESTED** [review]
Here are a few remarks.

File: src/mote/templates/mainpage.html [hunk 2]
```diff
@@ -32,13 +34,12 @@
         <link rel="icon" href="{{ url_for('static', filename='imgs/logoteal.png') }}" sizes="16x16">
         <script src="{{ url_for('static', filename='jscn/fragment.js') }}"></script>
         <script src="{{ url_for('static', filename='jscn/easy.qrcode.min.js') }}"></script>
-        <script src="{{ url_for('static', filename='jscn/socket.io.js') }}"></script>
-        <title>Fragment</title>
+        <title>MEETBOT LOGS</title>
```
Comment (darknao) [inline src/mote/templates/mainpage.html:37]: The all caps title seems a little bit aggressive to me.

**Comment by subhangi2731** [comment]
@darknao please check if everything is done

**Review by darknao — COMMENTED** [review]
Looks good :)
Please remove the two blank lines at the beginning of mainpage.html, then squash all your commits to one.

**Pull request closed without merging**